client/src/components/commentsEntry.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+// const CommentsEntry = ({comment}) => (
+//   <div>
+//     <div><img className="CM-profileImage" src={comment.user[0].image} alt="" /></div>
+//     <div className="CM-userName">{comment.user[0].name}</div>
+//     <div className="CM-commentText">{comment.text}</div>
+//     <div className="CM-time">{comment.time}</div>
+//   </div>
+// );
+
+class CommentsEntry extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      show: false,
+    };
+  }
+  showModal() {
+    this.setState({
+      show: !this.state.show,
+    })
+  }
+
+  render() {
+    return (
+      <div>
+        <div className='CM commentEntry'>
+        <div><img className="CM-profileImage" src={comment.user[0].image} alt="" /></div>
```

Comment:
not sure since can't see your data, but if you map the prop in commentsList, is there a reason you're only rendering image and name of first user in array (user[0])?

Reply:
the rest are going to be in a modal